packages/suite/src/actions/suite/analyticsActions.ts

```diff
@@ -521,22 +509,6 @@ export const init =
                 }),
             );
         });
-
-        // send OS type if isDesktop
-        if (isDesktop()) {
-            let desktopOSVersion = '';
-            const resp = await getOsType();
```

Comment:
remove also `async` keyword for the function, there is no more await

Reply:
Fixed and squashed